pkg/operator/configobservation/network/observe_network_test.go

```diff
@@ -3,18 +3,21 @@ package network
 import (
 	"testing"
 
+	"github.com/ghodss/yaml"
 	configv1 "github.com/openshift/api/config/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-
 	"k8s.io/client-go/tools/cache"
 
 	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
 	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
 	"github.com/openshift/library-go/pkg/operator/events"
+
+	. "github.com/onsi/gomega"
```

Comment:
use github.com/stretchr/testify/assert instead of this heavy weight

Reply:
done